websites/B/BookWalker/presence.ts

```diff
@@ -0,0 +1,74 @@
+const presence = new Presence({ clientId: "991729654757064774" });
+
+presence.on("UpdateData", async () => {
+	const presenceData: PresenceData = {
+		largeImageKey:
+			"https://pool.img.aptoide.com/split-store/85a754bbbfb92ee17701c03ad287ea68_icon.png",
+		startTimestamp: Math.floor(Date.now() / 1000),
```

Comment:
this should be declared as a constant outside the event, otherwise it won't work as expected